frontends/web/tsconfig.json

```diff
@@ -1,14 +1,16 @@
 {
     "compilerOptions": {
-        "target": "es5",
+        "target": "es2017",
```

Comment:
So, what's the difference... or rather, what are the advantages for bb app to switch the compiler target?

Reply:
One hope was that the build has real `class`, or that babel does not need to transpile `Object.assign` back to es5. Chromium should have close to 100% ES6+ compatibility. Last big open issue afaik was native `import` and `import()`. Qt chromium is probably a bit behind would be nice to have more info about that.

Another reason to up target was related to typescript
TypeScript complains about methods not existing on strings and arrays. 

btw.
The huge step in JavaScript was ES5 -> ES6 (which was renamed to es2015). From then  TC39 only included small stuff, afaik Array.prototype.includes, and in ES2017 async functions (I had to look that one up)

Reply:
Cool. Although, "hope" and "probably" are not very reassuring. I was hoping for a bit more affirmative statements. :)

Reply:
They are deliberately chosen, the build still does NOT use native `class`.

Reply:
> Qt chromium is probably a bit behind would be nice to have more info about that.

I'd love more info about that and shall gather info about chromium used in the Qt webengine